torch/distributed/rpc/__init__.py

```diff
@@ -26,9 +27,8 @@ def init_rpc(
         backend=backend_registry.BackendType.PROCESS_GROUP,
         init_method=None,
```

Comment:
Shouldn't init_method be subsumed by rpc_agent_options?

Reply:
@pritamdamania87 It's used by all backbends. So we don't want to put it into `RpcAgentOptions`.